apps/nextjs-example/babel.config.js

```diff
@@ -8,6 +16,7 @@ module.exports = {
       // https://stylexjs.com/docs/api/configuration/babel-plugin/
       {
         dev: process.env.NODE_ENV === 'development',
+        runtimeInjection: false,
```

Comment:
This is now always false because even in dev we use postcss to extract CSS statically